app/boards/shields/nice_view/nice_view.conf

```diff
@@ -0,0 +1,5 @@
+# Enable nice!view
+CONFIG_ZMK_DISPLAY=y
+CONFIG_SSD1306=n
```

Comment:
Why was this necessary? We used to have a bug or two with some random shield having conditionals wrong, causing this to auto-enable, but I don't think we still do?

Reply:
This was necessary as setting the `default` to `n` in the defconfig did not actually disable it since it's turned on in the other shield when `ZMK_DISPLAY` is enabled.